src/qml/FeatureForm.qml

```diff
@@ -300,7 +300,9 @@ Page {
             {
               AttributeValue = isNull ? undefined : value
               if ( qfieldSettings.autoSave && !dontSave ) {
-                save()
+                if ( ! save() ) {
+                  displayToast( qsTr( 'Unable to save changes') )
```

Comment:
I would prefer a log message than a display toast here.
Since it saves "in the background" it's very confusing for the user to get a toast.

Reply:
I reply here. First, thanks for the detailed review whether to show a toast or not. I think I've addressed all the comments below too. From now on I will keep in mind that background operations should not pop up toasts. As the log is already written withing the C++ part, I just need to revert the original QML most of the time.